SalesforceFoundation/EDA
# NPE Check on Affl deletion and new test method

# Critical Changes

# Changes
We've fixed a null pointer exception that occurs when a primary Affiliation with no Contact is deleted. With this change, EDA now deletes the Affiliation.

# Issues Closed
Fixes <a href="https://github.com/SalesforceFoundation/EDA/issues/834">#834</a>

# New Metadata

# Deleted Metadata

# Testing Notes
1) Create an account
2) Create a Contact
3) Create a primary affiliation between the account and the contact
4) Update the Affiliation Page layout by making the field 'Contact' not required
5) Clear the value in 'Contact' field
6) Delete the affiliation
The record should be deleted and no error messages like "There's a problem saving this record. You might not have permission to edit it, or it might have been deleted or archived. Contact your administrator for help." should be seen.
